WalletWasabi.Gui/CommandLine/Daemon.cs

```diff
@@ -13,89 +10,10 @@
 
 namespace WalletWasabi.Gui.CommandLine
 {
-	public static class Daemon
+	public class Daemon
 	{
-		public static async Task<bool> RunAsyncReturnTrueIfContinueWithGuiAsync(string[] args)
+		internal static async Task RunAsync(string walletName, LogLevel? logLevel, bool mixAll, bool keepMixAlive, bool silent)
 		{
-			var continueWithGui = true;
-			var silent = false;
-
-			var showHelp = false;
-			var showVersion = false;
-			LogLevel? logLevel = null;
-			string walletName = null;
-			var doMix = false;
-			var mixAll = false;
-			var keepMixAlive = false;
-
-			try
-			{
-				if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
This'll never run so I think you ruined the daemon in release too.

Reply:
What do you mean here? The referenced code segment was removed.

Reply:
Exactly.